dev.c

```diff
@@ -222,24 +231,15 @@ static void request_end(struct fuse_conn *fc, struct fuse_req *req,
 	if (shouldfree) fuse_request_free(req);
 }
 
-void fuse_request_send_nowait(struct fuse_conn *fc, struct fuse_req *req, bool force)
+void fuse_request_send_nowait(struct fuse_conn *fc, struct fuse_req *req)
```

Comment:
only allow queueing request to userspace if either of connected or allow_disconnected is true, even for fastpath requests. So removed force option.